app/javascript/mastodon/features/compose/components/search.tsx

```diff
@@ -506,7 +533,7 @@ export const Search: React.FC<{
 
       <ClearButton hasValue={hasValue} onClick={handleClear} />
 
-      <div className='search__popout'>
+      <div className='search__popout' tabIndex={-1}>
```

Comment:
This is needed so that focus doesn't automatically move to the `.ui` element when clicking anywhere in the popover.